python/hsfs/feature_group.py

```diff
@@ -1263,16 +1273,19 @@ def validate(
             self, dataframe, save_report, validation_options
         )
 
-    def compute_statistics(self, wallclock_time: Optional[str] = None):
+    def compute_statistics(
+        self, wallclock_time: Optional[Union[str, int, datetime, date]] = None
+    ):
         """Recompute the statistics for the feature group and save them to the
         feature store.
 
         Statistics are only computed for data in the offline storage of the feature
         group.
 
         # Arguments
-            wallclock_time: Date string in the format of "YYYYMMDD" or "YYYYMMDDhhmmss".
-                Only valid if feature group is time travel enabled. If specified will recompute statistics on
+            wallclock_time: datatime.datetime, datetime.date, unix timestamp in seconds (int), or string. The String should
+                be formatted in one of the following formats `%Y%m%d`, `%Y%m%d%H`, `%Y%m%d%H%M`, `%Y%m%d%H%M%S`,
```

Comment:
Why do we support these unreadable timestamp formats - not good for debugging. 
`%Y%m%d%H%M`
Can we also add common ones like ISO 86001
https://en.wikipedia.org/wiki/ISO_8601

`%Y-%m-%d %H:%M`

Reply:
ISO 8601 tackles this uncertainty by setting out an internationally agreed way to represent dates:

YYYY-MM-DD
https://www.iso.org/iso-8601-date-and-time-format.html

Reply:
This format is supported. We'd just need to update the docstrings, as discussed here https://github.com/logicalclocks/feature-store-api/pull/783#discussion_r976294576 There is also an open Jira, as part of which this schould be taken care of.